src/main/java/org/maxgamer/rs/core/Core.java

```diff
@@ -191,4 +191,8 @@ public static Timings getTimings() {
     public static ExecutorService getThreadPool() {
         return threadPool;
     }
+
+    private Core() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AS1213)